src/client/components/lazy-chat/readme.md

```diff
@@ -0,0 +1,16 @@
+# Lazy chat
```

Comment:
You have a nice idea about supporting different providers with standardised hook names. I'd document that here so that it doesn't get lost.